futures-channel/src/oneshot.rs

```diff
@@ -412,158 +411,3 @@ impl<T> Drop for Receiver<T> {
         self.inner.drop_rx()
     }
 }
-
-/// Handle returned from the `spawn` function.
-///
-/// This handle is a future representing the completion of a different future on
-/// a separate executor. Created through the `oneshot::spawn` function this
-/// handle will resolve when the future provided to `spawn` resolves on the
-/// `Executor` instance provided to that function.
-///
-/// If this handle is dropped then the future will automatically no longer be
-/// polled and is scheduled to be dropped. This can be canceled with the
-/// `forget` function, however.
-pub struct SpawnHandle<T, E> {
```

Comment:
Just to confirm, was this removed for "permanent removal" or just temporarily for executors to shake out?

Reply:
I think something like this should definitely exist, but it's going to depend on what happens w/ executors. Do you want me to leave a note about it somewhere?

Reply:
Ok yeah that's fine by me, want to add like a `TODO.md` or something like that in the repo which can be updated? (and mention this)

Reply:
Done.